File: HOME/Dz - 2/main1.py
#-------------Hw-2-#1----------------------------#
x = int(input("Введите число: "))
y = int(input("Введите число: "))
z = int(input("Введите число: "))

numbers = int(input("Выберите: \n1 - сложение \n2 - умножение"))
if numbers == 1:
    print(x+y+z)
elif numbers == 2:
    print(x*y*z)
else:
    print("Вы не выбрали ни один из вариантов")

File: HOME/Dz - 2/main1.js
x = +prompt("Введите число: ")
y = +prompt("Введите число: ")
z = +prompt("Введите число: ")

numbers = +prompt("Выберите: \n1 - сложение \n2 - умножение")
if (numbers == 1) {
    console.log(x+y+z)
} else if (numbers == 2) {
    console.log(x*y*z)
} else {
    console.log("Вы не выбрали ни один из вариантов")
}